### PR TITLE
Added `--ignore-unmonitored, -m` option to ignore any checks that are unmonitored which would normally trigger an unknown status

### PR DESCRIPTION
**Pull request by rwky**
## Pull Request Checklist

**Is this in reference to an existing issue?**

No

#### General

- [x] Update Changelog following the conventions laid out on [Keep A Changelog](http://keepachangelog.com/)

- [x] Update README with any necessary configuration snippets

- [x] Binstubs are created if needed

- [x] RuboCop passes

- [x] Existing tests pass

### CODE REVIEW

File: bin/check-monit-status.rb [hunk 1]
```diff
@@ -61,7 +67,7 @@ def run
       status = svc.elements['status'].text
 
       next if ignored.include? name
-
+      next if monitored == '0' && config[:ignore_unmonitored]
```
Comment (majormoses) [inline bin/check-monit-status.rb:70]: is this really the string `'0'`?

**Review by majormoses — COMMENTED** [review]
Seems legit, just want to confirm that the condition is correct as that seems fishy but I have seen similar terribleness with other services. Can I see an automated or manual [testing artifact](https://github.com/sensu-plugins/community/blob/master/PULL_REQUEST_PROCESS.md#7-testing-artifacts)?

**Comment by rwky** [comment]
Yep it's the string 0 I tested it on my end. The status codes aren't
documented you have to rummage in the monit source for them.

**Comment by rwky** [comment]
Re the testing artifact what exactly do you want? Writing an automated test would be a bit out of the scope here since there are no tests at all for this plugin. 

I could write tests for the entire plugin but not for a few weeks and it'd be best if that came in another PR.

**Comment by majormoses** [comment]
@rwky sorry this fell off my radar, just showing some simple input/output of the check showing it works is good enough for me.

**Comment by rwky** [comment]
I no longer use sensu, if someone else wants to take over this feel free.

**Pull request closed without merging**